content/docs/practices/histograms.md

```diff
@@ -0,0 +1,220 @@
+---
+title: Histograms and summaries
+sort_rank: 4
+---
+
+# Histograms and summaries
+
+Histograms and summaries are more complex metric types. Not only
+creates a single histogram or summary a multitude of time series, it
```

Comment:
"Not only creates a single histogram" is missing a subject, or at least it seems that way, because I think you mean "histogram or summary" to be the subject, but the location they appear in makes them seem like the object.

Probably you want to say "Not only do they create a multitude of time series, it is also ..."